pycbc/events/status.py

```diff
@@ -0,0 +1,252 @@
+# Copyright (C) 2018 Alex Nitz
+
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+#
+# =============================================================================
+#
+#                                   Preamble
+#
+# =============================================================================
+#
+""" Utitlities to query the status of gravitational-wave instruments
+from public sources as well as dqsegdb.
```

Comment:
This gives me a misleading impression of what this does. I read this and think that I can query (in realtime) if instruments are operational or not. Most of the stuff here is a little more technical (parsing veto-definers/querying segments). Also think the name "status" is confusing/misleading.

... It also doesn't make sense to have this in the "events" package. This is segments querying so we should use our name for this ("segments" ... or even "flags", but have this in a "segments" module, or a top level "segments.py")